setup.py

```diff
@@ -115,6 +115,7 @@
         'aenum',
         'typing;python_version<"3.5"',
         'windows-curses;platform_system=="Windows"',
+        'filelock'
```

Comment:
filelock should be in neovi under `extras_require`, right?

Reply:
Trying to keep this change as close as possible to the original PR https://github.com/hardbyte/python-can/pull/621.

Please open an issue if you think this should be move under `extras_require` and we can address it in 4.x.

Reply:
And by the way you are the one that removed it from `extras_require` on develop.
https://github.com/hardbyte/python-can/pull/833

Reply:
Yes, i'm aware of that :)

It was both in `extras_require` and `install_requires` so i wrongly assumed it was used elsewhere without checking it. We should avoid installing unnecessary packages though so i would prefer it in `neovi`.

Reply:
Done